scripts/config.js

```diff
@@ -15,26 +15,30 @@
  */
 
 const typescript = require('rollup-plugin-typescript2')
-const commonjs = require('rollup-plugin-commonjs')
-const replace = require('rollup-plugin-replace')
-const resolve = require('rollup-plugin-node-resolve')
+const commonjs = require('@rollup/plugin-commonjs')
+const replace = require('@rollup/plugin-replace')
+const { nodeResolve } = require('@rollup/plugin-node-resolve')
 const { uglify } = require('rollup-plugin-uglify')
 
 const packageDeps = require('../package.json').dependencies || {}
 const packagePeers = require('../package.json').peerDependencies || {}
 
 function getExternals(externals) {
+  if(externals === 'forBrowsers') {
+    return ['react']
+  }
   return externals === 'peers'
     ? Object.keys(packagePeers)
     : Object.keys(packageDeps).concat(Object.keys(packagePeers))
 }
 
-function getPlugins(env) {
+function getPlugins(env, externals) {
   const plugins = [
-    resolve(),
+    nodeResolve({
+      browser: externals === 'forBrowsers',
+    }),
     commonjs({
       include: /node_modules/,
-      namedExports: { '@optimizely/js-sdk-logging': ['getLogger'] },
```

Comment:
the cjs+esm interop with the older versions of rollup weren't working - when I added the umd+system builds, I also had to add a bunch of stuff to the `namedExports` field for various dependencies.

however, by updating to the latest version of rollup and commonjs plugin, they now handle that interop much better automatically and we can actually just remove this field all together.